lib/vmdb/loggers/container_logger.rb

```diff
@@ -1,61 +0,0 @@
-module Vmdb::Loggers
-  class ContainerLogger < VMDBLogger
```

Comment:
I just realized that now the ManageIQ::Loggers::Container logger is not the same as a VMDBLogger, and might lose the Q-task-id stuff - I'm not sure and I think I'd like to write some specs.

Reply:
Helper links for myself: 😄 

https://github.com/ManageIQ/manageiq-loggers/blob/master/lib/manageiq/loggers/base.rb#L134
vs
https://github.com/ManageIQ/manageiq-gems-pending/blob/master/lib/gems/pending/util/vmdb-logger.rb#L173
vs
https://github.com/ManageIQ/manageiq-loggers/blob/master/lib/manageiq/loggers/container.rb#L47

Reply:
Should this be WIP until then?

Reply:
Everything looks the same.  I noticed container logger doesn't have truncation of the message, but it didn't here either, so that matches.